assets/src/admin.scss

```diff
@@ -4,29 +4,37 @@
 
 body {
 	//group Information (optional) gray
-	h2 small, h3 small { 
+	h2 small, h3 small {
```

Comment:
looks like most of the changes here are from prettier (which is great) but i wonder if we could commit the whitespace changes to `develop` first and then that would help us see just the lines changed?